### ISSUE TITLE
Remove incorrect contributor

### ISSUE THREAD

**Issue by jakebrehm** [jakebrehm/drawing-epicycler#1]

**Comment by jakebrehm** [comment]
I changed the author and committer for each repository to myself using the following command:

```
git filter-branch -f --env-filter \
"GIT_AUTHOR_NAME='Jake Brehm'; GIT_AUTHOR_EMAIL='jake.m.brehm@gmail.com'; \
GIT_COMMITTER_NAME='Jake Brehm'; GIT_COMMITTER_EMAIL='jake.m.brehm@gmail.com';" HEAD
```

This corrected some old commits that I made from a different email address, but unfortunately this did not remove Jacob Alcott as a contributor despite him not being listed as making any contributions. He is also not showing up when using `git log`.

![Image](https://github.com/jakebrehm/drawing-epicycler/assets/33804275/e23d9807-9c97-405d-bc68-a37ea1ee7c10)

![Image](https://github.com/jakebrehm/drawing-epicycler/assets/33804275/266b2d7c-33d2-4628-ac37-ec3e33e84fe3)

![Image](https://github.com/jakebrehm/drawing-epicycler/assets/33804275/4996f350-29c3-478a-880f-d3480e9b250b)

I am closing this issue for now as I don't know of any other way to resolve it.
